browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -654,7 +654,7 @@ until they verify, or until 90 days have passed.
 	Pair
       </message>
       <message name="IDS_BRAVE_SYNC_CATEGORIES_TEXT" desc="Text for button to show data types to sync. Renamed from Categories">
-  Data Preferences
+ Sync Options 
```

Comment:
Thanks.
Please remove the trailing space and make the indentation as on other strings.
```
       <message name="IDS_BRAVE_SYNC_CATEGORIES_TEXT" desc="Text for button to show data types to sync. Renamed from Categories">
- Sync Options 
+       Sync Options
       </message>
```

Reply:
updated 👍